src/chunktransformer.ts

```diff
@@ -91,10 +92,15 @@ export class ChunkTransformer extends Transform {
     super(options);
     this.options = options;
     this._destroyed = false;
+    this._userCanceled = false;
     this.lastRowKey = undefined;
     this.reset();
   }
 
+  get canceled() {
```

Comment:
It seems like we expose ChunkTransformer as part of our public api. Which I think is incorrect to begin with. To minimize further leakage of implementation details, can we mark this method as internal?